app/src/main/java/com/example/criminalintentapp/presentation/authentication/Result.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.criminalintentapp.presentation.authentication
```

Comment:
Please create a `utils` package and place `Result.kt` and `AppModule.kt` inside it

Reply:
Just noticed there is also a `PictureUtils`, you can move that one as well inside the utils package